.github/workflows/pika.yml

```
@@ -188,8 +189,10 @@ jobs:
 
       - name: Cleanup
         run: |
-          rm -rf  ./deps
-          rm -rf ./buildtrees
+          cp deps/lib/libz.1.dylib .
+          cp deps/lib/libz.1.dylib tests/integration/
+          rm -rf ./deps 
```

yamllint

[error] trailing spaces

(trailing-spaces)